_source/logzio_collections/_log-sources/azure-blob-trigger.md

```diff
@@ -0,0 +1,208 @@
+---
+title: Ship logs from Azure Blob Storage using Azure Blob Trigger
+logo:
+  logofile: azure-blob.svg
+  orientation: vertical
+data-source: Azure Blob Storage via trigger
+templates: ["azure-deployment"]
+open-source:
+  - title: logzio-azure-blob-trigger
+    github-repo: logzio-azure-blob-trigger
+contributors:
+  - savidov
+  - nshishkin
+shipping-tags:
+  -  azure
+order: 460
+---
+
+<!-- tabContainer:start -->
+<div class="branching-container">
+
+* [Overview](#overview)
+* [Create a new blob storage account](#new-blob-config)
+* [Connect to existing blob storage account](#existing-blob-config)
+{:.branching-tabs}
+
+<!-- tab:start -->
+<div id="overview">
+
+Deploy this integration to forward logs from your Azure Blob Storage account to Logz.io using an automated deployment process via the trigger function. Each new log in the container path inside the storage account (including sub directories), will trigger the Logz.io function that will ship the file content to Logz.io.
+
+This integration offers the option to connect to an existing Blob storage account or create a new one, and it can be updated post-deployment.
+
+
+
+## Architecture overview
+
+The following services are created when you deploy this integration:
```

Comment:
Please change this line to:
The following resources that are needed for this integration
or something like that :)